config/dispatcher/dispatcher.ex

```diff
@@ -1,134 +1,140 @@
 defmodule Dispatcher do
-  use Plug.Router
+  use Matcher
 
-  def start(_argv) do
-    port = 80
-    IO.puts "Starting Plug with Cowboy on port #{port}"
-    Plug.Adapters.Cowboy.http __MODULE__, [], port: port
-    :timer.sleep(:infinity)
+  define_accept_types [
+    json: [ "application/json", "application/vnd.api+json" ],
+    html: [ "text/html", "application/xhtml+html" ],
+    any: [ "*/*" ]
+  ]
+
+  define_layers [ :static, :web_page, :api_services, :not_found ]
+
+  @api %{ layer: :api_services }
+
+  # Option calls are accepted by default
+  options "/*_path", _ do
+    conn
+    |> Plug.Conn.put_resp_header( "access-control-allow-headers", "content-type,accept" )
+    |> Plug.Conn.put_resp_header( "access-control-allow-methods", "*" )
+    |> send_resp( 200, "{ \"message\": \"ok\" }" )
   end
 
-  plug Plug.Logger
-  plug :match
-  plug :dispatch
+  get "/assets/*path", %{ layer: :static } do
+    forward conn, path, "http://mandaten/assets/"
+  end
 
-  # In order to forward the 'themes' resource to the
-  # resource service, use the following forward rule.
-  #
-  # docker-compose stop; docker-compose rm; docker-compose up
-  # after altering this file.
-  #
-  # match "/themes/*path" do
-  #   Proxy.forward conn, path, "http://resource/themes/"
-  # end
+  match "/files/*path", %{ layer: :static } do
```

Comment:
Moved it here since otherwise the frontend would be hit instead. It seems the browser also sends html accept headers when trying to download files.